src/openforms/js/components/admin/form_design/registrations/shared/OptionsConfiguration.js

```diff
@@ -16,6 +16,7 @@ const OptionsConfiguration = ({
   modalTitle,
   initialFormData,
   onSubmit,
+  modalSize = 'large',
```

Comment:
`modalSize` should probability be added to the propTypes :)
I'm seeing that `extraModifiers` is also missing from `FormModal.js`, maybe also add that one?